ocs/tests/test_defaults.py

```diff
@@ -1,7 +1,14 @@
 from ocs import defaults
+from tests import helpers
 
 
 def test_yaml_to_dict():
-    assert defaults.CEPHFILESYSTEM_DICT['apiVersion'] == 'ceph.rook.io/v1'
-    assert defaults.CEPHFILESYSTEM_DICT['spec']['metadataPool']['replicated']['size'] == 3
-    assert defaults.CSI_PVC_DICT['spec']['accessModes'] == ['ReadWriteOnce']
+    assert helpers.get_crd_dict(
+        defaults.CEPHFILESYSTEM_YAML
+    )['apiVersion'] == 'ceph.rook.io/v1'
```

Comment:
Use ocs.defaults.ROOK_API_VERSION

Reply:
I just reordered according to the coding guidelines.
Don't want to change functionality in this PR